packages/cf-component-progress/src/index.js

```diff
@@ -1,3 +1,6 @@
-import Progress from './Progress';
+import ProgressUnstyled from './Progress';
+import ProgressTheme from './ProgressTheme';
 
-export default Progress;
+const Progress = applyTheme(ProgressUnstyled, ProgressTheme);
+
+export { Progress as default, ProgressUnstyled, ProgressTheme };
```

Comment:
Oh, the toggle slipped through. All other migrated components don't export default afaik. I think it's better to grep&replace it in our codebase and keep it unified across the library so we don't have to go to documentation every single time to see how to import the component. It's going to be a major release anyway.

Sooner or later you might add another component into this package and then you have a problem because it's not obvious what the default is (think about having Button and a few months later adding ButtonGroup).